localization/CMakeLists.txt

```diff
@@ -1,35 +1,37 @@
-set(diagnostic_witness "${CMAKE_BINARY_DIR}/share/swift/diagnostics/generated")
+if(SWIFT_NATIVE_SWIFT_TOOLS_PATH)
```

Comment:
Please don't do this. Do this at the higher level and just avoid entering the directory.